packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.scss

```diff
@@ -0,0 +1,11 @@
+.wallets-jurisdiction-modal {
+    position: relative;
+    height: 80vh;
+    width: 85vw;
+    perspective: 200rem;
+
+    @include mobile {
+        height: auto;
+        width: 100%;
+    }
+}
```

Comment:
Is this styling only for dynamic leverage screen? If it is imo you should put this in the dynamic leverage screen styling, or into the screen component itself

Reply:
Only the screen should control the width/height of the modal imo

Reply:
Updated.